qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -178,6 +176,122 @@ public void test70BindMountCustomPathConfAndJvmOptions() throws Exception {
         }
     }
 
+    /**
+     * Check that environment variables can be populated by setting variables with the suffix "_FILE",
+     * which point to files that hold the required values.
+     */
+    public void test80SetEnvironmentVariablesUsingFiles() throws Exception {
+        final Path secretsDir = getTempDir().resolve("secrets");
```

Comment:
I would pull this into an instance variable and use `@Before` and `@After` to set it and clean up the dir. 
It's more readable and less repetitive avoiding the try, finally is particularly beneficial imho.